### PR TITLE
Two IT with different Byzantine behaviours

### PR DESCRIPTION
**Pull request by TheresePetersson**
* Integration test 1: Byz node sends two different views to different nodes, in order to try to trick some to migrate to a view change.

* Integration test2: Byz node sends RST_PAIR all the time to try to force a reset.

### CODE REVIEW

File: modules/view_establishment/module.py [hunk 1]
```diff
@@ -205,6 +205,33 @@ def send_msg(self):
                               self.witnesses[node_j],
                               self.pred_and_action.get_info(node_j)
                               ]
+
+                # Overwriting own_data to send different views to different
+                # nodes, to trick them
+                # if acting Byzantine with different_views - behaviour
+                if (byz.is_byzantine() and
```
Comment (axelniklasson) [inline modules/view_establishment/module.py]: Nit: Might make sense to do 

```
if byz.is_byzantine():
    if (byz.get_behavior() == byz.DIFFERENT_VIEWS:
        ...
    elif byz.get_behavior() == byz.FORCING_RESET:
        ...
```

this indicates that only one byz behavior is valid at a given point in time. not a big deal, just makes it a bit clearer when reading the code

File: tests/integration_tests/test_byz_node_sending_different_views.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+"""
+Case 7.1
+Byzantine node 0 tells some nodes that its view is (1,1) and others that it is (2,2).
```
Comment (axelniklasson) [inline tests/integration_tests/test_byz_node_sending_different_views.py]: Might be nice to describe the target state a bit here as well, i.e. that all nodes with `ID > 0` reach `{"current": 1, "next": 1}` and why that is the target state. Think we should document all integration tests more thoroughly as well, adding ticket for that

File: tests/integration_tests/test_nodes_cannot_progress_with_silent_nodes.py [hunk 3]
```diff
@@ -50,7 +50,7 @@
     }
 }
 
-class TestNodesConvergeThroughResetAll(AbstractIntegrationTest):
+class TestByzNodeSilent(AbstractIntegrationTest):
```
Comment (axelniklasson) [inline tests/integration_tests/test_nodes_cannot_progress_with_silent_nodes.py:53]: Nice 🍾

**Review by axelniklasson — APPROVED** [review]
Awesome work! Added some minor comments, but it looks good overall! 🇨🇾

**Comment by axelniklasson** [comment]
Oh and I approved this PR from the plane ✈️ I guess remote work really *flies*

File: modules/view_establishment/module.py (hunk 1 above)
Comment (TheresePetersson) [reply modules/view_establishment/module.py]: Of course!

File: tests/integration_tests/test_byz_node_sending_different_views.py (hunk 2 above)
Comment (TheresePetersson) [reply tests/integration_tests/test_byz_node_sending_different_views.py]: You are correct, updated the description in both tests.

**Pull request merged**